IABTechLab/uid2-self-serve-portal
# Remove approvers table

What Changed:

- Migration to delete Approvers table, plus remove code relating to approvers table or approvers mock data
- update `getParticipantsAwaitingApproval` to no longer use email or participant types since only uid2 support will see this
- Update the new participant request email to go to those with uid2support role, rather than being an approver
- Update ReadME when setting up portal 

Test Plan:
- Run this query in prod and integ to verify that all approvers have been migrated over as users. 
```
SELECT 
    a.email
FROM 
    approvers a
LEFT JOIN 
    users u ON a.email = u.email
LEFT JOIN 
    usersToParticipantRoles upr ON u.id = upr.userId AND upr.userRoleId = 3
WHERE 
    upr.userId IS NULL;
```

- Verify that in the Approved Participants table, the correct approver information still is displayed
- Verify that all participants with awaiting approval status are showing in the Manage Participants table
- Test sending a new participant request and make sure it goes to the emails of UID2 support role 
- Test that the new query in the ReadME for the user support role works
